nuitka/build/SconsInterface.py

```diff
@@ -163,7 +163,8 @@ def _setupSconsEnvironment():
         if os.getcwd() != os.getcwdu():
             os.chdir(getWindowsShortPathName(os.getcwdu()))
 
-    if Utils.isWin32Windows():
+    from nuitka.Options import shallUseStaticLibPython
```

Comment:
This is not going to work well, Options need to be passed via command line, as the options object is not going to be there, you are getting some sort of default here. But it's already passed and available as the `static_libpython` variable, so just use that please.

Reply:
Please clarify where I can find the `static_libpython` variable from here.

Reply:
Oh, that is me confused, I somehow thought this is in the scons file.